get-matrix.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+MIN_VERSION=$([[ $(cat appinfo/info.xml) =~ $(echo 'min-version=[^[0-9]]*([0-9]+).*') ]] && echo ${BASH_REMATCH[1]})
+MAX_VERSION=$([[ $(cat appinfo/info.xml) =~ $(echo 'max-version=[^[0-9]]*([0-9]+).*') ]] && echo ${BASH_REMATCH[1]})
+
+BASE_TEMPLATE='[VERSIONS]'
+VERSION_TEMPLATE='"NEXTCLOUD_VERSION"'
+
+if [ -z "$MIN_VERSION" ] && [ -z "$MAX_VERSION" ]; then
+    echo '["29"]'
+    exit 0
```

Comment:
I don't like having "the current version" defined in too many places - that gets annoying on upgrades and I personally tend to forget to update this stuff. Can't we just error here? Either there's something defined in info.xml or something is broken?

Reply:
Yeah you are right. I might be better to get notified when the constraints are empty.